crates/ra_project_model/src/cargo_workspace.rs

```diff
@@ -21,7 +21,7 @@ use crate::Result;
 pub struct CargoWorkspace {
     packages: Arena<Package, PackageData>,
     targets: Arena<Target, TargetData>,
-    pub(crate) workspace_root: PathBuf,
+    pub workspace_root: PathBuf,
```

Comment:
This is another crate, so let's be paranoid and add a `() -> &Path` getter. 

I don't really like when only some fields are `pub`, because that most often means that you are trying to protect "some" invariants, but you are probably allowing to  break others

Reply:
other than this, r=me

Reply:
Sounds reasonable, will address :+1:

Reply:
Also, we really should have an :bulb: for "generate getter"